webargs/core.py

```diff
@@ -363,9 +370,13 @@ def parse_arg(self, name, argobj, req, locations=None):
                 else:
                     value = argobj.default
             if argobj.required:
-                raise RequiredArgMissingError(
-                    'Required parameter "{0}" not found.'.format(name), arg_name=name
-                )
+                req_type = (argobj.required).__class__.__name__
```

Comment:
I'll take a look at this, the problem with `RequiredArgMissingError` is that the same use-case gets handled in two places.